docs/serving/autoscaling.md

```diff
@@ -1,16 +1,17 @@
 ---
-title: "Configuring the Autoscaler"
+title: "Autoscaling "
 weight: 10
 type: "docs"
 ---
 
 Since Knative v0.2, per revision autoscalers have been replaced by a single
```

Comment:
All references to old versions should be removed. Going forward, we should only reference `v.latest -1` re: https://github.com/knative/docs/issues/1850